R-package/README.md

```diff
@@ -300,8 +300,9 @@ At build time, `configure` will be run and used to create a file `Makevars`, usi
     ```shell
     docker run \
         -v $(pwd):/opt/LightGBM \
+        -w /opt/LightGBM \
         -t ubuntu:20.04 \
-        /bin/bash -c "cd /opt/LightGBM && ./R-package/recreate-configure.sh"
+        ./R-package/recreate-configure.sh
```

Comment:
Is it working without `/bin/bash`?

Reply:
yep! The file is executable and has the shebang `#!/bin/bash` at the top. https://github.com/microsoft/LightGBM/blob/dac774935cbadf4082bdc431b18b96c3fa21c8b1/R-package/recreate-configure.sh#L1

I tested this before pushing the PR, so I'm confident in it.